x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/Security.java

```diff
@@ -468,6 +468,12 @@ public class Security extends Plugin
         License.OperationMode.ENTERPRISE
     );
 
+    public static final LicensedFeature.Momentary USER_PROFILE_COLLABORATION_FEATURE = LicensedFeature.momentary(
+        null,
+        "user-profile-collaboration",
```

Comment:
The feature is named `user-profile-collaboration` instead of just user-profile because many of the basic APIs are not still Basic licensed and only the APIs relevant to the collaboration feature is protected. I am also open to other suggestions of names.